src/main/java/KeywordException.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+/**
+ * Encapsulates information and state of an EmptyException.
+ * Arises when user enters words that do not correspond to any command.
+ */
+public class KeywordException extends DukeException {
+    /** Fixed message to remind user to include keyword. */
+    static final String message = ":( OOPS! i'm sorry but i don't know what that means.";
```

Comment:
Same as before on constant names. Also I didn't mention previously but maybe could private the message since theres no need for any client to use this message outside of the class.